src/utils/share/index.js

```diff
@@ -73,7 +77,8 @@ const ShareUtils = {
   sharePostInTopic,
   shareUserLink,
   shareCommunity,
-  copyToClipboard
+  copyToClipboard,
+  copyTextToClipboard
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `copyTextToClipboard` has been added to the exported `ShareUtils` object. However, it's not clear from the provided context whether this function is used elsewhere in the codebase. If it's not used, it might be unnecessary to export it. Please ensure that all exported functions are actually needed and used.
```diff
  sharePostInTopic,
  shareUserLink,
  shareCommunity,
  copyToClipboard,
- copyTextToClipboard
+ // Ensure this function is used before exporting
+ // copyTextToClipboard
};

export default ShareUtils;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->